.azure-pipelines/compliance/CredScanSuppressions.json

```diff
@@ -30,16 +34,20 @@
             "_justification": "No need to scan VS Code test folder."
         },
         {
-            "folder": "ui\\.vscode-test\\",
+            "folder": "utils\\.vscode-test\\",
```

Comment:
Out of curiosity, did CredScan actually find something there?

Reply:
Good point, it hasn't. Should I remove that until it does?

Reply:
I vote yes. Can't hurt.